src/icons/LifeBuoy/LifeBuoy.native.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
LifeBuoy?? 🤣

Reply:
Picked it from the figma file. Didn't know what else to call it 😅